lib/index.config.js

```diff
@@ -23,36 +23,20 @@ export default function routes($ocLazyLoadProvider, $stateProvider, $translatePr
 		url: '/page1',
 		views: {
 			mainView: {
-				templateProvider: ($q) => {
-					return $q((resolve) => {
-						// lazy load the view
-						require.ensure([], () => resolve(require("./components/page1/page1.html")));
-					});
-				},
-				controller: function () {
-					this.name = 'Hello Page1';
-				}
+				component: 'page1'
 			},
 			rightMenuView: {
-				templateProvider: ($q) => {
-					return $q((resolve) => {
-						// lazy load the view
-						require.ensure([], () => resolve(require("./components/rightMenu/rightMenu.html")));
-					});
-				}
+				component: 'rightMenu'
 			}
 		},
+		resolvePolicy: {
+			module: {when: 'EAGER'}
+		},
 		resolve: {
-			loadPage1Component: ($q, $ocLazyLoad) => {
-				return $q((resolve) => {
-					require.ensure([], () => {
-						// load whole module
-						let module = require('./components/page1/page1.module');
-						$ocLazyLoad.load({name: 'page1'});
-						resolve(module.component);
-					});
-				});
-			}
+			module: ($q, $ocLazyLoad) => $q(resolve =>
+				require.ensure(['components/page1/page1.module'], require =>
```

Comment:
Webpack 1 syntax is not great.
The syntax in webpack version 2 is much better, and uses `System.import`